.github/workflows/winget-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Submit Winget Package to Windows Package Manager Community Repository
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "Specific tag name"
+        required: false
+        type: string
+env:
+  CI: true
+jobs:
+  winget-publish:
+    name: Publish winget package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set winget version env
+        env:
+          TAG_NAME: ${{ inputs.tag_name || github.event.release.tag_name }}
+        run: echo "WINGET_TAG_NAME=$(echo ${TAG_NAME#v})" >> $GITHUB_ENV
```

Comment:
what does this do exactly ?

Reply:
Stores into an env variable either the name provided by the user firing the github action manually, or the one provided by the automated workflow run on release.

Reply:
my question was specifcally about the `${TAG_NAME#v}` . Why is there a `v` here ?

Reply:
That's a good question, not sure about that. From what I've read, if there is a prefix v1.2.3 in version tag name, then this gets auto stripped by the github action. Seems that this is what this #v does.